web/source/text/prediction/modelManager.ts

```diff
@@ -374,7 +374,7 @@ namespace com.keyman.text.prediction {
     }
 
     public get enabled(): boolean {
-      return this._mayPredict;
+      return this.activeModel && this._mayPredict;
```

Comment:
In the Android emulator, (default SIL  Eurolatin), I'm not getting suggestions.

Remote into the [WebView](chrome://inspect/#devices), I get `keyman.modelManager.activeModel` is `undefined `

Reply:
Right, shouldn't have pushed it out quite that quickly - the enablement logic needed a bit of extra work, as I accidentally left it without a way to actually activate a model.  The new commit fixes that.